packages/opentelemetry-sampler-aws-xray/src/remote-sampler.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Sampler,
+  SamplingDecision,
+  SamplingResult,
+} from '@opentelemetry/sdk-trace-base';
+import { diag } from '@opentelemetry/api';
+import { SamplingRule, AWSXRaySamplerConfig } from './remote-sampler.types';
+import axios from 'axios';
+
+const DEFAULT_INTERVAL_MS = 5 * 60 * 1000; // 5 minutes on sampling rules fetch (default polling interval)
+const DEFAULT_ENDPOINT = 'http://localhost:2000';
+const SAMPLING_RULES_ENDPOINT = '/GetSamplingRules';
+
+// IN PROGRESS - SKELETON CLASS
+export class AWSXRayRemoteSampler implements Sampler {
+  private _pollingInterval: number;
+  private _endpoint: string;
+
+  constructor(samplerConfig: AWSXRaySamplerConfig) {
+    this._pollingInterval =
+      samplerConfig.pollingInterval ?? DEFAULT_INTERVAL_MS;
+    this._endpoint = samplerConfig.endpoint
```

Comment:
nit: here you are concating something to a variable named "endpoint" and storing the result value in another variable called "endpoint". This is not very clean and can cause confusion in the future.

I think if you give them better names, it would increase readability. Things like:
"DEFAULT_AWS_BASE_ENDPOINT", "SAMPLING_RULES_URL_PATH", "samplingRulesFullEndpoint". You can also come up with other options. For me, it's not easy to read something like: `this._endpoint = samplerConfig.endpoint + SAMPLING_RULES_ENDPOINT`, as everything is named "endpoint" and the context is only in how it is used and not in the variable name.